### ISSUE TITLE
Add flag to disable auto generation of mapper classes.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vadbch/orika#22]
```
For security / paranoia reasons I would like to scope the mapping to classes 
that I have manually defined mappers for.  Basically the factory should refuse 
to generate a mapper in lookupmapper if one has not been defined already.  I've 
attached a patch built against 1.2.0 that shows the behavior I am trying to 
achieve.
```

Original issue reported on code.google.com by `ransombr...@gmail.com` on 31 May 2012 at 2:38

Attachments:
- [patch](https://storage.googleapis.com/google-code-attachments/orika/issue-22/comment-0/patch)

**Comment by GoogleCodeExporter** [comment]
```
Ok, an initial implementation of this is available in the 1.2.0 branch.
It's not exactly as your patch (instead call useAutoMapping(false) on the 
Builder), as I factored it into part of a change to make DefaultMapperFactory 
more extensible.

Please check it out and let me know if this is working as expected.
```

Original comment by `matt.deb...@gmail.com` on 1 Jun 2012 at 5:15
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Sorry for the delay, but this is working as expected.

Thanks,
Ransom
```

Original comment by `ransombr...@gmail.com` on 6 Jun 2012 at 2:57

**Comment by GoogleCodeExporter** [comment]
```
Excellent; thanks.
Marked as fixed in the 1.2.0 branch
```

Original comment by `matt.deb...@gmail.com` on 9 Jul 2012 at 1:59
- Changed state: **Fixed**
